stubs/Markdown/markdown/util.pyi

```diff
@@ -41,15 +41,15 @@ class HtmlStash:
 
 class Registry:
     def __init__(self) -> None: ...
-    def __contains__(self, item): ...
+    def __contains__(self, item: Any): ...
```

Comment:
Literally any object can be passed into `Registry.__contains__` without it failing, so `object` is a better annotation than `Any` here. Also, the return type can be guaranteed to be `bool`.

https://github.com/Python-Markdown/markdown/blob/12c3378ab9a46fae416a7bb5549e3f123a11d22a/markdown/util.py#L302-L307

```suggestion
    def __contains__(self, item: object) -> bool: ...
```